var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -541,13 +523,35 @@ def configure(self, spec, prefix):
             )
             )
 
-    def build(self, spec, prefix):
-        python('./x.py', 'build', extra_env={
+    def install(self, spec, prefix):
```

Comment:
This change includes a couple small changes to the Rust package:
1) Starts building Rust with optimizations for the target microarchitecture
2) Removes the separate build phase and collapses it all into install.